pep-0649.rst

```diff
@@ -573,25 +601,25 @@ Language Reference:
     annotations values should be provided.  Must be one of the
     following:
 
-    ``1`` (exported as ``inspect.VALUE``)
+    ``inspect.VALUE`` (equivalent to the ``int`` constant ``1``)
```

Comment:
If we are putting these in an enum, should they really be aliased as top-level attributes of `inspect`? It always seemed slightly odd to me in terms of namespacing, since there are a ton of things in `inspect` that have nothing to do with annotations. Is it that painful for someone wanting to use these values to do `from inspect import AnnotationFormat` and then `AnnotationFormat.VALUE`?

Reply:
I think so.  `AnnotationFormat` is a lot to type.  Let me put it this way: this is a common enough idiom that enum explicitly supports it, via `@enum.global_enum`.

As a guy who uses `inspect.signature` more than the average bear, I often alias `inspect.Parameter.POSITIONAL_OR_KEYWORD` precisely because of how much *#%@! typing that is.

Reply:
I believe `@global_enum` mostly exists for backward compatibility with previous module-level names. I won't insist on it but I'd also vote to get rid of the global names and use only the enum.